src/org/scada_lts/web/security/XssValidatorException.java

```diff
@@ -0,0 +1,25 @@
+package org.scada_lts.web.security;
```

Comment:
As I said, you throw it into the package:
org.scada_lts.web.beans.validation.xss

There are validations using the Java Bean Validation API mechanism.